cli/cmd/inject.go

```diff
@@ -386,4 +387,5 @@ func init() {
 	injectCmd.PersistentFlags().UintSliceVar(&ignoreOutboundPorts, "skip-outbound-ports", nil, "outbound ports that should skip the proxy")
 	injectCmd.PersistentFlags().UintVar(&proxyControlPort, "control-port", 4190, "proxy port to use for control")
 	injectCmd.PersistentFlags().UintVar(&proxyAPIPort, "api-port", 8086, "port where the Conduit controller is running")
+	injectCmd.PersistentFlags().StringVar(&proxyLogLevel, "log-level", "warn,conduit_proxy=info", "log level for the proxy")
```

Comment:
Yeah, so the way the `RUST_LOG` environment variable works is that it can take a comma separated list of { `level` | `scope` | `scope=level` }. A bare log `level` will enable that level for all modules, a bare `scope` will enable all log levels for that scope, and `scope=level` will set the log level for that scope to the given level. 

Valid log levels (see https://docs.rs/log/0.4.1/log/#use) are: 
- `trace`
- `debug`
- `info`
- `warn`
- `error`

The scopes we're probably the most likely to care about are:
- `conduit_proxy`: the proxy itself
- `h2`, `tower-h2`: the libraries we use for HTTP/2
- `hyper`: the library we used for HTTP/1
- `tower-grpc`: the gRPC library (probably will want the `h2` libs as well for debugging gRPC issues)

You can set more granular scopes as well, but this is based on the layout of the Rust modules in the proxy. 

For more in depth docs, see https://doc.rust-lang.org/log/env_logger/index.html#enabling-logging

Reply:
Hmm, it occurs to me that this wall of text could probably go in docs somewhere now that I think of it.